benchmark/vm_ivar_get.yml

```diff
@@ -0,0 +1,37 @@
+prelude: |
+  class Example
+      def initialize
+          @v0 = 1
+          @v1 = 2
+          @v3 = 3
+          @levar = 1
+      end
+
+      def get_value_loop
+          sum = 0
+
+          i = 0
+          while i < 1000000
+              # 10 times to de-emphasize loop overhead
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              sum += @levar
+              i += 1
+          end
+
+          return sum
+      end
+  end
```

Comment:
Can we consistently use 2-space indentation? We used to use 4 spaces in some benchmarks in yjit-bench, but we recently made them consistent by using 2 spaces too. I think other existing benchmarks in `benchmark/` also use 2 spaces.